argusclient/client.py

```diff
@@ -591,6 +591,72 @@ def get_alerts_allinfo(self, ownerName=None, alertNameContains=None, shared=Fals
+        return self.argus._request("get", uri_path)
+
+    def create_composite_alert(self, payload):
+        if not payload: raise ValueError("Need to specify an payload")
+        alert_obj = self.argus._request("post", "alerts", dataObj=payload)
+        self._coll[alert_obj.id] = alert_obj
+        return alert_obj
+
+    def add_child_alert_to_composite_alert(self, alert_id, payload):
+        if not alert_id: raise ValueError("Need to specify composite alert id")
+        if not payload: raise ValueError("Need to specify  payload")
+        uri_path = "alerts/{}/children".format(alert_id)
+        alert_obj = self.argus._request("post", uri_path, dataObj=payload)
+        self._coll[alert_obj.id] = alert_obj
+        return alert_obj
+
+    def update_composite_alert(self, alert_id, payload):
+        if not alert_id: raise ValueError("Need to specify composite alert id")
+        if not payload: raise ValueError("Need to specify  payload")
+        uri_path = "alerts/{}".format(alert_id)
+        return self.argus._request("put", uri_path, dataObj=payload)
+
+    def delete_child_alert_from_composite_alert(self, comp_alert_id, child_alert_id):
+        """
+        Delete a child alert from a composite alert
+        """
+        if not comp_alert_id: raise ValueError("Need to specify composite alertid")
+        if not child_alert_id: raise ValueError("Need to specify a child alertid")
+        uri_path = "alerts/{}/children/{}".format(comp_alert_id, child_alert_id)
+        if id in self._coll:
+            del self._coll[child_alert_id]
+        return self.argus._request("delete", uri_path)
+
+    def add_trigger_to_child_alert(self, alert_id, trigger):
+        if not alert_id: raise ValueError("Need to specify alertid")
+        if not isinstance(trigger, Trigger): raise TypeError("Need a Trigger object, got: %s" % type(trigger))
+        uri_path = "alerts/{}/triggers".format(alert_id)
+        return self.argus._request("post", uri_path, dataObj=trigger)
+
+    def del_trigger_from_child_alert(self, alert_id, trigger_id):
+        if not alert_id: raise ValueError("Need to specify alert id")
+        if not trigger_id: raise ValueError("Need to specify trigger id")
+        uri_path = "alerts/{}/triggers/{}".format(alert_id, trigger_id)
+        return self.argus._request("delete", uri_path)
+
+    def add_notification_to_composite_alert(self, alert, notification):
+        if not isinstance(alert, Alert): raise TypeError("Need a Alert object, got: %s" % type(alert))
+        if not isinstance(notification, Notification): raise TypeError(
+            "Need a Notification object, got: %s" % type(notification))
+        uri_path = "alerts/{}/notifications".format(alert.id)
+        notification_obj = self.argus._request("post", uri_path, dataObj=notification)
+        return notification_obj
+
+    def del_notification_from_composite_alert(self, alert, notification_id):
```

Comment:
Duplicate of `alert.notifications.delete()`?

Reply:
Fixed code

Reply:
I think this method too should be removed, but I started a new thread here: https://github.com/salesforce/python-argusclient/pull/28#discussion_r634080461